Testing/Exec/IgnitionDelay/computeIgnitionDelay.py

```diff
@@ -72,33 +72,33 @@ def readMultiColFile(filename,headerSize=0):
     group2.add_argument('-v','--verbose', action='store_true', help='verbose')
     group2.add_argument('-q','--quiet', action='store_true', help='quiet')
     args = parser.parse_args()
-    
-    
+
+
     # Read Pele input file
     inpt = parseInputFile(args.inputFile)
     dt = float(inpt['ode.dt'])
     ndt = float(inpt['ode.ndt'])
     press = float(inpt['hr.press'])
     temp = float(inpt['hr.t0'])
     phi = float(inpt['hr.equiv_ratio'])
-    
-    
+
+
     # Process result
     filePP = 'PPreaction.txt'
     A1 = readMultiColFile(filePP)
     ignitionDelayPP = (dt/ndt)*np.amin(np.argwhere(A1[:,1]>(A1[0,1]+400)))
-    
-    
+
+
     if args.estimate:
-        if args.verbose: 
-            logMessage = f"Rough estimate: T = {temp:.3g} K , P = {press/101325.0:.3g} atm, Phi = {phi:.3g},  tIgn = {ignitionDelayPP:.3g} +/- {dt/ndt:.3g} s"
+        if args.verbose:
+            logMessage = f"Rough estimate: T = {temp:.3g} K , P = {press/1013250.0:.3g} atm, Phi = {phi:.3g},  tIgn = {ignitionDelayPP:.3g} +/- {dt/ndt:.3g} s"
             print(logMessage)
         # Write new input file
         writeNewInput(args.inputFile,'inputs/inputs.0d_refine',ignitionDelayPP*1.1,ndt*2)
-    
-    if args.refine: 
-        logMessage = f"T = {temp:.3g} K , P = {press/101325.0:.3g} atm, Phi = {phi:.3g},  tIgn = {ignitionDelayPP:.3g} +/- {dt/ndt:.3g} s"
+
+    if args.refine:
+        logMessage = f"T = {temp:.3g} K , P = {press/1013250.0:.3g} atm, Phi = {phi:.3g},  tIgn = {ignitionDelayPP:.3g} +/- {dt/ndt:.3g} s"
         print(logMessage)
-        writeResult('log',temp,press,phi,ignitionDelayPP,dt/ndt) 
+        writeResult('log',temp,press,phi,ignitionDelayPP,dt/ndt)
```

Comment:
Could you please check if the 101325 factor is also messed up in `writeResult`?

Reply:
Could call, it also needed correction there.